docs/guides/bulk/download.md

```diff
@@ -0,0 +1,32 @@
+---
+heading: Bulk
+seo: Bulk | Bulk Download | How to Use Bulk Download
+title: Download
+description: How to use Bulk Download
+layout: sidebarleft
+apis: API Docs
+breadcrumbs: /docs/guides/home.html
+parent: Back to Guides
+order: 2
+---
+
+# Bulk Download
+
+The Bulk download can be used to generate a csv or json file from a large number of records. Since the bulk download works asynchronously, two calls are required to download a file.
```

Comment:
Use bulk download to generate a csv or json file from a large number of records. Since the bulk download works asynchronously, you need to make two calls to download a file.

- Making it active